### PR TITLE
Create dockershim shortcode

### PR DESCRIPTION
**Pull request by chrisnegus**
Creating a shortcode to put on pages that mention "Docker Engine" to alert people of the Dockershim removal and point them to the "Dockershim Removal FAQ", per [Issue #32805](https://github.com/kubernetes/website/pull/32805).

I've not done one of these before, so I assume that I'll have to work with it for a bit to get it right before this PR is ready for review.

### CODE REVIEW

**Comment by chrisnegus** [comment]
Here is a list of documentation files in content/en/docs that include the term "Docker Engine":

```
./concepts/services-networking/service.md
./reference/command-line-tools-reference/feature-gates.md
./reference/glossary/docker.md
./reference/node/topics-on-dockershim-and-cri-compatible-runtimes.md
./setup/production-environment/container-runtimes.md
./tasks/administer-cluster/kubeadm/adding-windows-nodes.md
./tasks/administer-cluster/migrating-from-dockershim/migrating-telemetry-and-security-agents.md
./tasks/administer-cluster/migrating-from-dockershim/migrate-dockershim-dockerd.md
./tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md
./tasks/network/customize-hosts-file-for-pods.md
```

Should any of these include the dockershim note?

**Comment by chrisnegus** [comment]
Preview (see the note at the top of the page): https://deploy-preview-32826--kubernetes-io-main-staging.netlify.app/docs/setup/production-environment/container-runtimes/

**Comment by sftim** [comment]
@chrisnegus ideally, squash this down to 2 commits:
- adding the shortcode
- actually using the shortcode

That split will help localization teams

File: data/i18n/en/en.toml [hunk 1]
```diff
@@ -36,6 +36,9 @@ other = " documentation is no longer actively maintained. The version you are cu
 [deprecation_file_warning]
 other = "Deprecated"
 
+[dockershim_message]
+other = """The Dockershim code required to run Docker Engine has been removed from the Kubernetes project as of release 1.24. Read the <a href="/blog/2022/02/17/dockershim-faq/">Dockershim Removal FAQ</a> for further details."""
```
Comment (sftim) [inline data/i18n/en/en.toml:40]:
```suggestion
# Localization note: you can hyperlink to your localized version
# of https://k8s.io/blog/2022/02/17/dockershim-faq/ if you wish to.
[dockershim_message]
other = """The dockershim code required to run Docker Engine has been removed from the Kubernetes project as of release 1.24. Read the <a href="/dockershim">Dockershim Removal FAQ</a> for further details."""
```

**Review by sftim — COMMENTED** [review]
Just some thoughts

File: data/i18n/en/en.toml (hunk 1 above)
Comment (chrisnegus) [reply data/i18n/en/en.toml:40]: Okay. Done.

**Comment by sftim** [comment]
@chrisnegus is this ready for review?

**Comment by chrisnegus** [comment]
> @chrisnegus is this ready for review?

Sure. I'll remove the hold. I was going to also include the shortcode in a whole bunch of pages. But let's review this and get the shortcode merged. Then I'll add the shortcodes to the necessary pages in a separate PR.

**Comment by chrisnegus** [comment]
> @chrisnegus ideally, squash this down to 2 commits:
> 
> * adding the shortcode
> * actually using the shortcode
> 
> That split will help localization teams

Okay.

**Comment by chrisnegus** [comment]
Here you can see the shortcode showing up as a note at the top of the container-runtimes page.
dockershimShortCode

File: data/i18n/en/en.toml [hunk 2]
```diff
@@ -36,6 +36,9 @@ other = " documentation is no longer actively maintained. The version you are cu
 [deprecation_file_warning]
 other = "Deprecated"
 
+[dockershim_message]
+other = """The dockershim code required to run Docker Engine has been removed from the Kubernetes project as of release 1.24. Read the <a href="/dockershim">Dockershim Removal FAQ</a> for further details."""
```
Comment (reylejano) [inline data/i18n/en/en.toml:40]:
```suggestion
other = """The dockershim code required for Kubernetes to use Docker Engine has been removed from the Kubernetes project as of release 1.24. Read the <a href="/dockershim">Dockershim Removal FAQ</a> for further details."""
```

Comment (zacharysarah) [reply data/i18n/en/en.toml:40]: @reylejano I like this. I think we can refine it somewhat for clarity and let the FAQ do the heavy lifting.
```suggestion
other = """Dockershim has been removed from the Kubernetes project as of release 1.24. Read the <a href="/dockershim">Dockershim Removal FAQ</a> for further details."""
```

Comment (reylejano) [reply data/i18n/en/en.toml:40]: I prefer @zacharysarah 's suggestion over mine :)

Comment (chrisnegus) [reply data/i18n/en/en.toml]: Okay. I used @zacharysarah wording and did a new commit. Thanks!

**Pull request merged**